iphone/Maps/Core/Location/MWMLocationManager.mm

```diff
@@ -451,7 +469,8 @@ - (void)setStarted:(BOOL)started
     return;
   NSNotificationCenter * notificationCenter = NSNotificationCenter.defaultCenter;
   if (started) {
-    _started = [self start];
+    [self start];
+    _started = YES;
     if (_started) {
```

Comment:
1. Is this check needed?
2. What if [self start] fails later, should the following notificationCenter subscriptions stay active?

Reply:
I will bring back the returning BOOL in `[self start]` as `[CLLocationManager locationServicesEnabled]` to skip observing if needed but error processing will happen only in the delegate.
So we will check global system locationServicesEnabled twice:
1) one time when determine is location is working
2) how to handle this situation in the delegate